OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -305,7 +315,7 @@ public override bool HandleMouseInput(MouseInput mi)
 
 				if (mi.Event == MouseInputEvent.Up)
 				{
-					var wasJoystickScrolling = IsJoystickScrolling;
+					var wasJoystickScrolling = joystickScrollStart.HasValue && joystickScrollEnd.HasValue;
```

Comment:
This is the obvious cause for the breakage for joystick-scrolling. I simply changed the code to ignore the deadzone. Reverting the change to this line *will* re-introduce the bug from #11849 for joystick-scrolling.